draft-ietf-rtcweb-mdns-ice-candidates.md

```diff
@@ -234,10 +234,13 @@ as a tradeoff for improved peer-to-peer connectivity.
 #### mDNS Candidate Encoding
 
 The mDNS name of an mDNS candidate MUST be used in the connection-address field
-of its candidate attribute. When an mDNS candidate is the default candidate,
-its mDNS name MUST be used in the connection-address field of the SDP "c=" line.
-Since an mDNS candidate also conceals its address family, the "c=" line
-SHOULD use "IP4" in the address-type field.
+of its candidate attribute. However, when an mDNS candidate would be the
+default candidate, typically because there are no other candidates, its mDNS
+name MUST NOT be used in the connection-address field of the SDP "c=" line, as
+experimental deployment has indicated that many remote endpoints will fail to
+handle such a SDP. In this situation, the IP address "0.0.0.0" and port value
+"9" MUST instead be used in the c= and m= lines, similar to how the
+no-candidates case is handled in {{ICESDP}}, Section 4.3.1.
```

Comment:
The 'port value "9"' is somehow confusing since c= line will not include it?
Or is it supposed to impact other lines of the SDP?

Also, the spec currently says 'SHOULD use "IP4"'.
The PR does not have any such wording anymore.
It seems the change to use "0.0.0.0" explicitly leans twoards 'MUST use "IP4"' but this not clear.
At least in Safari, we might end up generating "c=IN IP6 ::" in cases the unobfuscated host IP address is IPv6.

Reply:
I tried to use similar language to that in https://tools.ietf.org/html/draft-ietf-mmusic-ice-sip-sdp-39#section-4.3.1, but open to suggestions.

Generally, I think we should standardize on 0.0.0.0:9 as that is more consistent with what ice-sip-sdp recommends. In this case, use of IP4 should be clear.

Reply:
https://tools.ietf.org/html/draft-ietf-mmusic-ice-sip-sdp-39#section-4.3.1 is mentioning "::", I would keep it, especially since Safari implementation may use it.
I would keep a MUST for "0.0.0.0" and "::".
I am ok with an explicit SHOULD to use IP4 (hence 0.0.0.0).

Reply:
Sorry I missed this one.

> a MUST for "0.0.0.0" and "::"
> an explicit SHOULD to use IP4.

This looks good to me.

Reply:
Will adjust accordingly

Reply:
PTAL

Reply:
LGTM modulo there is a line removed above, which looks accidental.